Proiect-VS/assets/css/evaluations.less

```diff
@@ -0,0 +1,91 @@
+body {
+    position: relative;
+    font-family: Arial;
+    background-color: #ffffff; 
+    margin: auto;
+}
+
+header {
+    height: 70px;
+    border-bottom: 1px solid grey;
+}
+
+footer {
+    height: 60px; 
+    position: absolute;
+    left: 0;
+    right: 0;
+    line-height: 60px;
+    background-color: #bcbcaf;
+    color: #686865;
+    text-align: right;
+    padding-right: 50px;
+
+}
+
+a {
+    vertical-align: middle;
+    line-height: 70px;
+    margin-left: 100px;
+    margin-right: 15px;
+    font-size: 25px;
+}
+
+.logo {
+    width: auto;
+    height: 50px;
+    line-height: 70px;
+    vertical-align: middle;
+}
+
+.candidates_table {
+    margin: 200px auto 440px;
+    width: 60%;
+    height: 200px;
+    border-collapse: collapse;
+    text-align: left;
+    font-size: 20px;
+}
+
+.candidates_table_header {
+    background-color: #21bcff;
+    border: none;
+    height: 60px;
+    color: white;
+    padding-left: 30px;
+}
+
+.current_page {
+    border-bottom: 9px solid #21bcff; 
```

Comment:
General comment: Colors should not be hard coded. Define them in a colors.less file and import them here.